fixtures/arri/l10-c.json

```diff
@@ -0,0 +1,393 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "L10-C",
+  "categories": ["Dimmer", "Color Changer"],
+  "meta": {
+    "authors": ["Dyami Caliri"],
+    "createDate": "2018-10-17",
+    "lastModifyDate": "2018-10-17"
+  },
+  "links": {
+    "manual": [
+      "https://www.arri.com/resource/blob/31168/649ca2ae2935bc75686e42fbe388e07f/l5-0006896-arri-l-series-l10-user-manual-en-nov2018-l03301-data.pdf"
```

Comment:
Can you please also add the DMX chart as manual (to all 3 fixtures)?
https://www.arri.com/resource/blob/65926/3b71e4fe69caea09f657fa8d599a55ab/l5-0022935-arri-l-series-dmx-protocol-specification-v3-4-data.pdf

This makes it easier to review if the DMX definitions are correct.